Packages/com.unity.inputsystem/CHANGELOG.md

```diff
@@ -44,6 +44,7 @@ however, it has to be formatted properly to pass verification tests.
 - Fixed InputManager.asset file growing in size on each Reset call.
 - Fixed Opening InputDebugger throws 'Action map must have state at this point' error
 - Fixed Cut/Paste behaviour to match Editor - Cut items will now be cleared from clipboard after pasting.
+- Improved initial window layout to avoid elements being hidden.
```

Comment:
I would suggest being more specific in the CHANGELOG about which window we are referring to, e.g. "Project Settings - Input Actions".